README.md

```diff
@@ -41,6 +41,21 @@ Chimera is currently composed of three Apache Camel Components, available as sub
 - `camel-chimera-rmlmapper` Camel component used to lifting using the [rmlmapper-cefriel](https://github.com/cefriel/rmlmapper-cefriel) library 
 - `camel-chimera-mapping-template` Camel component able to implement both lifting and lowering steps using the [mapping-template](https://github.com/cefriel/mapping-template) library
 
+### Integration with Camel Karavan
+
+[Camel Karavan](https://camel.apache.org/categories/Karavan/) is a visual editor for Apache Camel routes, facilitating the design, testing, and monitoring of integration flows. It provides a user-friendly interface for managing Camel routes and their associated components. Chimera components can be integrated into the [Visual Studio Code plugin for Camel Karavan](https://marketplace.visualstudio.com/items?itemName=camel-karavan.karavan) by following these steps.
+
+1. Install Visual Studio Code.
+2. Install the Karavan extension for Visual Studio Code.
+3. Navigate to the Karavan directory within the Visual Studio Code installation directory and locate the _components.json_ file.
+4. Copy the contents of the [graph.json](./camel-chimera-graph/src/generated/resources/com/cefriel/component/graph.json), [mapt.json](./camel-chimera-mapping-template/src/generated/resources/com/cefriel/component/mapt.json) and [rml.json](./camel-chimera-rmlmapper/src/generated/resources/com/cefriel/component/rml.json) files to the _components.json_ file. An example [components.json](./karavan/components.json) is provided, note however that this file has been only been tested with Karavan v4.4.0.
+5. Chimera components are now selectable in the Karavan Visual Studio Code UI.
+
+An example route built using this tool is available [here](https://github.com/cefriel/chimera-tutorial/tree/yaml-tutorial). Additionally, as of version
```

Comment:
Specify that currently some manual editing is needed to make the pipeline extracted from Karavan executable because custom components are not officially supported.